### ISSUE TITLE
[Bug]: After upgrade v1.35.1: Problems with Select component when based on a collection.

### ISSUE THREAD

**Issue by MGeurts** [tasteui/tasteui#616]
### What Happened

My Livewire component has this:
```
    public function genders(): Collection
    {
        return Gender::select('id', 'name')->orderBy('name')->get();
    }
```

and this:

```
<x-ts-select.styled wire:model="childForm.gender_id" id="gender_id" label="{{ __('person.gender') }} :" :options="$childForm->genders()" select="label:name|value:id"
                            placeholder="{{ __('app.select') }} ..." wire:dirty.class="bg-warning-200 dark:text-black" searchable />
```

The Select component now throws an error:

```
array_values(): Argument #1 ($array) must be of type array, Illuminate\Database\Eloquent\Collection given
```
It was working totaly fine in version 1.35.0. The error appeared in version 1.35.1.

### Component

Select Styled

### How to Reproduce

Use a Select component based on a model (Eloquent collection).

### Public Repository

https://github.com/MGeurts/genealogy

### Laravel Version

11.20

### Livewire Version

3.5

### TallStackUI Version

1.35.1

### PHP Version

8.3.10

### Operation System

Windows

**Comment by devajmeireles** [comment]
Fixed in v1.35.2. Thanks for reporting.

**Comment by MGeurts** [comment]
Thanks.
